roles/ceph-common/tasks/pre_requisites/prerequisite_rh_storage_iso_install.yml

```diff
@@ -12,11 +12,15 @@
     src: "{{ ceph_stable_rh_storage_iso_path }}"
     dest: "{{ ceph_stable_rh_storage_iso_path }}"
 
+# assumption: ceph_stable_rh_storage_mount_path does not specify directory
```

Comment:
so you're only concern is that we don't validate the existence of this directory?
I can do this in another patch if you don't have time, let me know :)